base-theme/layouts/partials/external-link.html

```diff
@@ -0,0 +1,13 @@
+{{- $text := default .title .text | htmlEscape | .context.RenderString -}}
+{{- $isImage := default false .is_image -}}
+{{- $href := default "" .external_url -}}
+
+{{- if and .is_broken .backup_url -}}
```

Comment:
Can we assign local variables for these and have defaults, like the other values above? Something like:

```
{{- $isBroken = default false .is_broken -}}
{{- $backupUrl default "" .backup_url -}}
```